test/test.py

```diff
@@ -220,7 +226,7 @@ def test_binaryen_from_source(self):
   def test_no_32bit(self):
     print('test 32-bit error')
     emsdk_hacked = hack_emsdk('not is_os_64bit()', 'True')
-    failing_call_with_output('python %s install latest' % emsdk_hacked, 'this tool is only provided for 64-bit OSes')
+    failing_call_with_output('python3 %s install latest' % emsdk_hacked, 'this tool is only provided for 64-bit OSes')
```

Comment:
Question about this bit: I updated this to python3 because just 'python' doesn't work on mac when updating to the newer XCode. But 'python3' doesn't work on windows.

what is this test trying to check? Is there a better way than this?

Reply:
I don't think that anything currently exists.. maybe prove using `shutil.which` to see which one to use (`https://docs.python.org/3/library/shutil.html#shutil.which)